src/preferences/dialog/dlgprefwaveformdlg.ui

```diff
@@ -473,20 +473,36 @@ Select from different types of displays for the waveform, which differ primarily
         <string/>
        </property>
        <property name="text">
-        <string>Beat Grid</string>
+        <string>Beat Grid Opacity</string>
```

Comment:
Only capitalize the first word to match the rest of the dialog: "Beat grid opacity"